runtime/doc/news.txt

```diff
@@ -38,6 +38,14 @@ The following new APIs or features were added.
 • |vim.iter()| provides a generic iterator interface for tables and Lua
 iterators |luaref-in|.
 
+• Added automatic linting of treesitter query files (see |ft-query-plugin|).
+  Automatic linting can be turned off via >vim
+    autocmd FileType *.scm let b:lint_on = []
```

Comment:
```suggestion
    autocmd FileType query let b:lint_on = []
```

Reply:
I would prefer Lua snippets in general (for consistency), although Vim is of course more concise here.

Reply:
still wanted to note that `autocmd FileType query let b:lint_on = []` is not working for me. If you have an alternative suggestion in Lua. I'll paste it here

Reply:
I'll test. (But this autocommand was wrong in any case, `*.scm` is not a valid FileType pattern.)